meta-mender-core/recipes-mender/mender-connect/mender-connect.inc

```diff
@@ -53,23 +47,22 @@ do_prepare_mender_connect_conf[vardeps] = " \
     MENDER_CONNECT_USER \
 "
 
-do_install() {
-    oe_runmake \
-        -C ${B}/src/${GO_IMPORT} \
-        V=1 \
-        prefix=${D} \
-        bindir=${bindir} \
-        datadir=${datadir} \
-        sysconfdir=${sysconfdir} \
-        systemd_unitdir=${systemd_unitdir} \
-        install-bin \
-        install-systemd
-
-    # install configuration
-    mkdir -p  ${D}/${sysconfdir}/mender
+# Set the version of mender-connect by passing it as an extra ldflag to go.bbclass
+# This is equivalent to how it's set in the Makefile
+GO_EXTRA_LDFLAGS:append = "-X github.com/mendersoftware/mender-connect/config.Version=$(git describe --tags --dirty --exact-match 2>/dev/null || git rev-parse --short HEAD)"
```

Comment:
(thinking out loud)

To be yocto-correct, we should use [PV](https://docs.yoctoproject.org/ref-manual/variables.html#term-PV) variable:

```suggestion
# This is equivalent to how it's set in the Makefile
GO_EXTRA_LDFLAGS:append = "-X github.com/mendersoftware/mender-connect/config.Version=${PV}"
```

However, this in effect will modify how the versions are seen for the `git` recipes, as there we would get something like version `master-git` instead of the Git sha. Then, to workaround this, we could modify *only for the git recipe* the GO_EXTRA_LDFLAGS to use the git command.

At the end of the day how it is right now represents better how it was behaving under the hood, so I think it is just fine :)